.eslintrc.json

```diff
@@ -90,6 +90,12 @@
           }
         ]
       }
+    },
+    {
+      "files": ["**/ci/*.js"],
+      "rules": {
+        "no-undef": "off"
```

Comment:
`process.env` is a global that do not work with specified eslint env

Reply:
it doesn't work and I think it is ok to change for build / ci scripts 
alternative is to enable eslint env to `node` which is not desired due to it not being our primary env

Reply:
exactly, that's what I did - enabled separate rule to `ci` scripts :)